apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -505,6 +532,94 @@ await connection.BeginReadOnlyTransactionAsync(TimestampBound.Strong))
                 }
             }
         }
+
+        [Fact]
+        public async Task ScopeCompleteWithTwoReadsDoesntThrow()
+        {
+            await WriteSampleRowsAsync();
+            using (var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
+            {
+                var readConnection1 = await _testFixture.GetTestDatabaseConnectionAsync();
+                var readConnection2 = await _testFixture.GetTestDatabaseConnectionAsync();
```

Comment:
Usings for these? (same comment in other 2 tests below). If you're worried about nesting by the way, you can stack them:
```csharp
using (var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
using (var readConnection1 = await _testFixture.GetTestDatabaseConnectionAsync())
using (var readConnection2 = await _testFixture.GetTestDatabaseConnectionAsync())
{
}
```